packages/react-core/src/core/providers/thirdweb-wallet-provider.tsx

```diff
@@ -227,6 +228,16 @@ export function ThirdwebWalletProvider(
     }
   }, [hiddenConnectionWallet]);
 
+  /**
+   * Clear the states of hidden connection
+   */
+  const disconnectHiddenConnection = useCallback(() => {
+    setSigner(undefined);
+    setActiveWallet(undefined);
+    setActiveWalletConfig(undefined);
+    setConnectionStatus("disconnected");
+  }, []);
```

Comment:
what is a hidden connection?

Reply:
@iketw - ignore this. I'm gonna completely change this